packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -151,6 +153,7 @@ export class ARRenderer extends EventDispatcher {
     // @TODO #293, handle WebXR API changes
```

Comment:
Have we handled this TODO yet? And any others in there for that matter?

Reply:
From what I've seen, webxr.ts already was fairly up-to-date, I'll make another pass at this file and try to remove the `any` casts - they are probably the best way to determine whether the API changes are already handled.

Reply:
This TODO is mostly resolved but I'd expect this to be an ongoing effort as WebXR is adding new features and typings will probably always be slightly behind it.